### PR TITLE
(maint) release prep for 1.0.x branch

### PR DESCRIPTION
**Pull request by sarameisburger**
whoops i put the 1.0.x stuff in the master branch changelog

### CODE REVIEW

**Comment by mwaggett** [comment]
I think future mergeups will be easier if you do a mergeup after your master changelog goes in and then add the changes for 1.0.x.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,30 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
...
+- Fail if nonfinal_repo_name is unset for nonfinal repos.
+- Added option for user to specify which directory of packages they want to sign.
+
+### Changed
+- Use `tar` labeled nodes to build tarballs
+- Use `ppc` in AIX artifact paths by updating the `parse_platform_tag` function.
+- Renamed redhat-fips platform to redhatfips.
+
+### Fixed
+- Fix nightlies path so we can pass a variable through when we're working on
```
Comment (mwaggett) [inline CHANGELOG.md]: Could this be more specific perhaps? What variable?

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,30 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
...
+a nightly ship.
+- Copy pasta error where we were syncing the latest filename instead of the directory.
```
Comment (mwaggett) [inline CHANGELOG.md]: I think this should be reversed. We were previously syncing the directory/path, but now we're just syncing the file.

**Comment by sarameisburger** [comment]
@mwaggett updated

**Comment by sarameisburger** [comment]
Updated the release date

**Comment by sarameisburger** [comment]
rebased and dates updated

**Pull request merged**